miio/tests/test_airhumidifier.py

```diff
@@ -3,14 +3,17 @@
 import pytest
 
 from miio import AirHumidifier
+from miio import AirHumidifierMjjsq
```

Comment:
Looks good to me, but I'm wondering if this should be in a separate file (considering it's a different product line)? Otherwise feel free to merge :tada:

Reply:
I will move it to a separate file.